src/components/AppMenu.tsx

```diff
@@ -68,7 +68,7 @@ export function AppMenu() {
           </MenuItem>
         </Menu>
       </div>
-      <PreferencesDialog open={openPreferences} onClose={handleClosePreferences} />
+      {openPreferences && <PreferencesDialog open={openPreferences} onClose={handleClosePreferences} />}
```

Comment:
Given it's not going to be loaded at all if openPreferences is false, does the "open" property need to exist on PreferencesDialog anymore?

Reply:
`open` is a required prop on Dialog